src/sage/misc/cachefunc.pyx

```diff
@@ -2640,13 +2654,17 @@ cdef class CachedMethod():
             sage: a.f0()
             4
 
-        The computations in method ``f`` are tried to store in a
-        dictionary assigned to the instance ``a``::
+        For methods with parameters, computations in method ``f`` are
```

Comment:
for methods with parameters the result of method f is attempted to be stored in a dictionary attribute of the instance ``a''